plugins/generic/usageStats/UsageStatsOptoutBlockPlugin.inc.php

```diff
@@ -16,130 +16,91 @@
 import('lib.pkp.classes.plugins.BlockPlugin');
 
 class UsageStatsOptoutBlockPlugin extends BlockPlugin {
-
-	/** @var string */
-	var $_parentPluginName;
-
+	/** @var UsageStatsPlugin Parent plugin */
+	protected $_parentPlugin;
```

Comment:
Minor comment: `parent` might be confusing terminology here, because of it's use to describe class relationships (`parent::method()`). Maybe just refer to it as `usageStatsPlugin`?

Reply:
will do...